src/main/java/org/synyx/urlaubsverwaltung/absence/DateRange.java

```diff
@@ -1,18 +1,34 @@
 package org.synyx.urlaubsverwaltung.absence;
 
+import org.springframework.util.Assert;
+
 import java.time.LocalDate;
 import java.util.Iterator;
 import java.util.NoSuchElementException;
 import java.util.Objects;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
+import static java.time.temporal.ChronoUnit.DAYS;
+
+/**
+ * Represents a immutable date range
+ * <p>
+ * A date range represents a period of time between two LocalDates.
+ * Date range are inclusive of the start and the end date.
+ * The end date is always greater than or equal to the start date.
+ * <p>
+ */
 public final class DateRange implements Iterable<LocalDate> {
```

Comment:
What was the reason to not use/depend on javas `Period`?

Reply:
I dont know yet, i used it because we already had it and we decided to also usw it as global class...maybe we need to take a deeper look what is the benefit of DateRange

Reply:
Period is old and DateRange is the new kid on the block of the new AbsencePeriod